Content.Server/Projectiles/Components/HitscanComponent.cs

```diff
@@ -21,22 +22,17 @@ namespace Content.Server.Projectiles.Components
     public class HitscanComponent : Component
     {
         [Dependency] private readonly IGameTiming _gameTiming = default!;
-        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
 
         public override string Name => "Hitscan";
+        public CollisionGroup CollisionMask => (CollisionGroup) _collisionMask;
 
         private TimeSpan _startTime;
         private TimeSpan _deathTime;
 
         [DataField("layers")] //todo  WithFormat.Flags<CollisionLayer>()
         private int _collisionMask = (int) CollisionGroup.Opaque;
         [DataField("damage")]
-        private float _damage = 10f;
-
-        [DataField("damageType", required: true)]
-        private string _damageTypeID = default!;
-
-        private DamageTypePrototype _damageType => _prototypeManager.Index<DamageTypePrototype>(_damageTypeID);
+        public float Damage { get; set; } = 10f;
```

Comment:
should be int?

Reply:
The damage is sometimes scaled in `ServerBatteryBarrelComponent`, if the laser weapon doesn't have enough charge for a full shot. I'm guessing this is why it was made a float? It's currently rounded to an int in `ServerRangedBarrelComponent`. For now I'd just leave it as is as it'll be changed whenever damage units are added anyways.